### DISCUSSION TITLE
Error from API: You need to sign in or sign up before continuing

### DISCUSSION THREAD

**Discussion by davydov87** [chatwoot/chatwoot#5270] [Chatwoot APIs] [answered]
Hello!

I want to use the "List all inboxes" method, but I can't get authorized. 

I get an error from api: You need to sign in or sign up before continuing.

I add in the header parameter " api_access_token" and the value - my token from the profile. 

Is this not correct?

![image](https://user-images.githubusercontent.com/111266847/184583449-134ecf12-0473-426c-bcee-12a275fd24df.png)

**Comment by bernapastorini** [comment]
Hello! 
Have you solved the problem? If not, please post some more context (ex: the entire request).

**Comment by aapeliv** [comment, marked as answer]
For posterity: I had this same problem. It was caused by nginx dropping the `api_access_token` header because it uses underscores. The fix was to add `underscores_in_headers on;` in the `server` block in nginx config. See the nginx config in the repo:

https://github.com/chatwoot/chatwoot/blob/4ed13787d1d359eb3b70c828afb890b0b58c96c8/deployment/nginx_chatwoot.conf#L28

**Comment by WillyMaikowski** [comment]
For posterity: I had this problem and I found that services like Cloudflare "filter" this header inconsistently (sometimes appears, sometimes it was filtered). I tested defining more details in the access log of the nginx to identify the irregularity of the "api_access_token". From there, It was only a thing from my architecture so I tested eliminating CF and It works.
